recipes-support/librealsense/librealsense_1.10.0.bb

```diff
@@ -88,3 +88,10 @@ FILES_${PN}-tests += "\
 
 # Configuration is not required
 do_configure[noexec] = "1"
+
+#Skip ldflags QA check
```

Comment:
Yeah. It seems librealsense/Makefile does not use LDFLAGS at all. There's a PR pending to move to CMake but that does not take LDFLAGS either.

Looks like this is the only way until librealsense itself is fixed.